llvm/lib/Target/Sparc/SparcInstrInfo.td

```diff
@@ -693,38 +693,38 @@ let DecoderNamespace = "SparcV8", Predicates = [HasNoV9] in {
 }
 
 let rd = 0 in {
-  let Defs = [CPSR] in {
-    def STCSRrr : F3_1<3, 0b110101, (outs (MEMrr $rs1, $rs2):$addr), (ins),
+  let mayStore = 1, Uses = [CPSR] in {
```

Comment:
The mayStore setting looks like an unrelated change

Reply:
> The mayStore setting looks like an unrelated change

I agree but I don't want those instructions are optimized away after this change. I cannot find any test from sparc target. Not sure whom should we ask for help.

Reply:
But can you precommit them? That should be doable without the operand change?